primitives/manta/src/assets.rs

```diff
@@ -352,6 +352,7 @@ where
         asset_id: AssetId,
         account: &C::AccountId,
         amount: Balance,
+        mint: bool,
```

Comment:
Please add to the docstring what `mint` does and rename it to a boolean name like `should_mint` or `is_mint` whatever the appropriate thing is.

Reply:
https://github.com/Manta-Network/Manta/pull/571/files#diff-e7fdf4a079f6a2130e9cc7ef71f550c7ab4bdfe6f3b85a6a0f1ad8a1863091ceR327